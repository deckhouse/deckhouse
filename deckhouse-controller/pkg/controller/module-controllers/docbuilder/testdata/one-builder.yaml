apiVersion: deckhouse.io/v1alpha1
kind: ModuleDocumentation
metadata:
  name: testmodule
spec:
  version: v1.0.0
  checksum: abc
  path: /testmodule/v1.0.0
---
apiVersion: coordination.k8s.io/v1
kind: Lease
metadata:
  labels:
    deckhouse.io/documentation-builder-sync: ""
  name: module-docs-builder-ldq7x
  namespace: d8-system
spec:
  holderIdentity: 10-111-2-176.d8-system.pod.cluster.local:8081
  leaseDurationSeconds: 35
  renewTime: "2024-05-08T12:11:04.160683Z"
