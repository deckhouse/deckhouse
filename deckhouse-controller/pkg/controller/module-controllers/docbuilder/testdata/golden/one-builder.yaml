---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleDocumentation
metadata:
  creationTimestamp: null
  name: testmodule
  resourceVersion: "1000"
spec:
  checksum: abc
  path: /testmodule/v1.0.0
  version: v1.0.0
status:
  conditions:
  - address: http://10-111-2-176.d8-system.pod.cluster.local:8081
    checksum: abc
    lastTransitionTime: "2019-10-17T15:33:00Z"
    message: ""
    type: Rendered
    version: v1.0.0
  result: Rendered
